src/Microsoft.OData.Core/ODataMessageWriterSettings.cs

```diff
@@ -367,19 +377,24 @@ internal SelectedPropertiesNode SelectedProperties
         }
 
         /// <summary>
-        /// Func to evaluate whether an annotation should be written by the writer. The func should return true if the annotation should
+        /// Func to evaluate whether an annotation should be written by the writer based on the annotations filtered in the odata.include-annotations header and OData standard.
+        /// The func should return true if the annotation should
         /// be written and false if the annotation should be skipped.
         /// </summary>
-        internal Func<string, bool> ShouldIncludeAnnotation
+        /// <remarks>
+        /// This property is internal and automatically set based on annotation filters. The developer can override this using the <see cref="ShouldIncludeAnnotation"/> property

```

Comment:
Is it really overriden?

Reply:
I've removed the word `override` to avoid any confusion